src/ingest_validation_tools/submission.py

```diff
@@ -79,12 +81,19 @@ def get_errors(self):
         return errors
 
     def _get_plugin_errors(self):
-        plugin_path = Path(__file__).parent / PLUGIN_DIR_REL_PATH
-        errors = defaultdict(list)
-        for metadata_path in self.effective_tsv_paths.values():
-            for k, v in run_plugin_validators_iter(metadata_path, plugin_path):
-                errors[k].append(v)
-        return {k: v for k, v in errors.items()}  # get rid of defaultdict
+        plugin_path = self.plugin_dir_abs_path
+        if plugin_path:
+            errors = defaultdict(list)
+            for metadata_path in self.effective_tsv_paths.values():
+                try:
+                    for k, v in run_plugin_validators_iter(metadata_path,
+                                                           plugin_path):
+                        errors[k].append(v)
+                except PluginValidatorError as e:
+                    errors['Unexpected Plugin Error'] = str(e)
```

Comment:
This can be hit multiple times, but you'll only keep the last error. Maybe build a list of all errors?

Reply:
(But, related to the points above, I'd prefer that the plugins not be run when their are more basic issues with the submission... so I wouldn't focus work here.)

Reply:
This error only occurs on a malfunction, rather than in response to errors in the actual submission.  Thus I thought that a single exception condition would suffice.